articles/connections/passwordless/regular-web-app-email-link/v7/index.md

```diff
@@ -3,7 +3,7 @@ title: Using Passwordless Authentication with a magic link via email on Regular
 ---
 # Passwordless Authentication with a magic link via e-mail on Regular Web Apps
 
-<%= include('../../_older-version-alert' %>
+<%= include('../../../../_includes/_version_warning_auth0js' %>
```

Comment:
The warning says that they should migrate to Auth0.js v9 asap, but [the snippet already uses v9](https://auth0-docs-content-pr-5709.herokuapp.com/docs/connections/passwordless/regular-web-app-email-link/v7#use-your-own-ui)

Reply:
I also think this is confusing one way or another. What if someone does not use the custom UI option and he sees this panel telling him to upgrade to Auth0.js v9? For mixed docs (those that include both Lock and Auth0.js) the message should be different (or it should be a `warning` under each respective title)